src/test/resources/docker-compose.yml

```
@@ -16,3 +16,48 @@ services:
     - "35672:5672"
     - "46671:15671"
     - "46672:15672"
+
+ redis:
+  container_name: redis-case-it
+  image: redis:3.2.9
+  ports:
+  - "17379:6379"
+
+ iac:
+  container_name: iac-case-it
+  image: eu.gcr.io/census-rm-ci/rm/census-rm-iacsvc
+  ports:
+  - "18121:8121"
+  - "15121:5121"
+  - "18221:8221"
+  external_links:
+  - postgres-case-it
+  - rabbitmq-case-it
+  - redis-case-it
+  environment:
+  - SECURITY_BASIC_ENABLED=true
+  - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres-case-it:5432/postgres?sslmode=disable
+  - LIQUIBASE_URL=jdbc:postgresql://postgres-case-it:5432/postgres
+  - RABBITMQ_HOST=rabbitmq-case-it
+  - RABBITMQ_PORT=5672
+  - REDIS_HOST=redis-case-it
+  - CASE_SVC_CONNECTION_CONFIG_HOST=foobar
+  - CASE_SVC_CONNECTION_CONFIG_PORT=666
+  - JAVA_OPTS=-Xmx128m -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5121
```

```suggestion
  - JAVA_OPTS=-Xmx128m -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=5121 -Dspring.profiles.active=dev
```
Is now required to have default config.